### PR TITLE
Wrap closure linter

### PR DESCRIPTION
**Pull request by kewisch**

### CODE REVIEW

File: lib/ical/property.js [hunk 1]
```diff
@@ -161,25 +161,27 @@ var ICAL = ICAL || {};
...
-      if(aName == "VALUE") {
+      if (aName == "VALUE") {
         this.data.type = aValue;
         // TODO revalidate value
       }
     },
 
     countParameters: function countParmeters() {
-      return(this.data.parameters ? Object.keys(this.data.parameters).length : 0);
+      // TODO Object.keys compatibility?
```
Comment (lightsofapollo) [inline lib/ical/property.js:171]: There are many ES5 compat libs as along as we don't do anything crazy thats a fallback option. I thought that Object.keys was there in IE8 though...

Comment (kewisch) [inline lib/ical/property.js:171]: This page says IE9: https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Object/keys
The workaround on that page looks quite ugly (dontEnumBug)

File: lib/ical/value.js [hunk 2]
```diff
@@ -142,13 +143,14 @@ var ICAL = ICAL || {};
       //if (typeof this.window['btoa'] == 'function') {
       //    return btoa(data);
       //}
-      var b64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```
Comment (lightsofapollo) [inline lib/ical/value.js:145]: Maybe we can put these in helper.js and use the browser version when available? (General comment not for this review)

File: tools/icaljs_errorrules.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+def InjectErrorReporter():
+  global OriginalShouldReportError
+  OriginalShouldReportError = errorrules.ShouldReportError
+  errorrules.ShouldReportError = IcalShouldReportError
+
+def IcalShouldReportError(error):
+  global OriginalShouldReportError
+  return error not in (
+    errors.UNNECESSARY_DOUBLE_QUOTED_STRING,
```
Comment (lightsofapollo) [inline tools/icaljs_errorrules.py:18]: Very cool!!!

**Pull request merged**